### PR TITLE
Updates prepping for julia 0.7

### PR DESCRIPTION
**Pull request by TransGirlCodes**
WIP

### CODE REVIEW

**Review by bicycle1885 — COMMENTED** [review]
I personally don't like inserting spaces around `=`. Keyword argument without spaces is the convention in the Python world. If you don't have a strong opinion here, please keep it as is.

File: src/Automa.jl [hunk 1]
```diff
@@ -3,6 +3,15 @@ __precompile__()
 module Automa
 
 import DataStructures: DefaultDict
+import Compat: Compat,
```
Comment (bicycle1885) [inline src/Automa.jl:6]: I think you need to bump the Compat.jl version.

File: REQUIRE [hunk 2]
```diff
@@ -1,3 +1,3 @@
 julia 0.6
 DataStructures 0.5.1
-Compat 0.41.0
+Compat
```
Comment (bicycle1885) [inline REQUIRE]: You need to specify the lower bound here. I think v0.55 is fine.

**Comment by TransGirlCodes** [comment]
Ok I think this is ready now. The rest of the syntax warnings seem to be from non Automa places.

**Review by bicycle1885 — APPROVED** [review]

**Comment by bicycle1885** [comment]
Great work! Thank you very much!

**Pull request merged**